awx/ui_next/src/screens/Template/shared/SurveyToolbar.test.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { mountWithContexts } from '@testUtils/enzymeHelpers';
+import { act } from 'react-dom/test-utils';
+import SurveyToolbar from './SurveyToolbar';
+
+jest.mock('@api/models/JobTemplates');
+
+describe('<SurveyToolbar />', () => {
+  test('delete Button is disabled', async () => {
+    let wrapper;
+
+    await act(async () => {
+      wrapper = await mountWithContexts(
+        <SurveyToolbar
```

Comment:
- The component under test doesn't do any asynchronous loading, so you don't need `await`, `act`, or `async` in these tests. Please change this and other tests in this file to setup tests like this:
```jsx
  test('delete Button is disabled', () => {
    const wrapper = mountWithContexts(
      <SurveyToolbar
        isDeleteDisabled
        onSelectAll={jest.fn()}
        isAllSelected
        onToggleDeleteModal={jest.fn()}
        onToggleSurvey={jest.fn()}
      />
    );
   ...
})
```